src/inference/dev_api/openvino/runtime/system_conf.hpp

```diff
@@ -157,9 +189,43 @@ enum ColumnOfProcessorTypeTable {
     PROC_TYPE_TABLE_SIZE = 4   //!< Size of processor type table
 };
 
+/**
+ * @enum       ProcessorUseStatus
+ * @brief      Definition of CPU_MAP_USED_FLAG column in CPU mapping table.
+ */
+enum ProcessorUseStatus {
+    NOT_USED = -1,           //!< Processor is not bound to thread
+    CPU_USED = 1,            //!< CPU is in using
+    PLUGIN_USED_START = 100  //!< Plugin other than CPU needs to use. If more GPUs use CPUs, the CPU_MAP_USED_FLAG is
+                             //!< accumulated from PLUGIN_USED_START. For example: GPU.0:100, GPU.1:101
+};
+
+/**
+ * @brief      Get and reserve available cpu ids
+ * @ingroup    ie_dev_api_system_conf
+ * @param[in]  core_type core type.
+ * @param[in]  num_cpus number of cpus.
+ * @param[in]  seek_status look for CPU_MAP_USED_FLAG of seek_status in CPU mapping table
+ * @param[in]  reset_status reset CPU_MAP_USED_FLAG with reset_status.
+ * @return     Array of available cpu ids.
+ */
+OPENVINO_RUNTIME_API std::vector<int> reserve_available_cpus(const ColumnOfProcessorTypeTable core_type,
+                                                             const int num_cpus,
+                                                             const int seek_status = NOT_USED,
+                                                             const int reset_status = CPU_USED,
+                                                             const bool reserve_logic_core = false);
+
+/**
+ * @brief      Set CPU_MAP_USED_FLAG of cpu_mapping
+ * @ingroup    ie_dev_api_system_conf
+ * @param[in]  cpu_ids cpus in cpu_mapping.
+ * @param[in]  used update CPU_MAP_USED_FLAG of cpu_mapping with this flag bit
+ */
+OPENVINO_RUNTIME_API void set_cpu_used(std::vector<int>& cpu_ids, int used);
```

Comment:
```suggestion
OPENVINO_RUNTIME_API void set_cpu_used(const std::vector<int>& cpu_ids, int used);
```
?

Reply:
I think it is no need to add `const`, because cpu_ids is modified in this function.